src/client/trackers/pageAttention.js

```diff
@@ -0,0 +1,32 @@
+import { broadcast } from '../broadcast';
+import ScrollDepth from './page-attention/ScrollDepth';
+import AttentionTime from './page-attention/AttentionTime';
+
+export const pageAttention = (options = {}) => {
+	const onExit = (attentionTime) => {
+		broadcast('oTracking.event', {
+			category: 'page',
+			action: 'interaction',
+			context: {
+				attention: {
+					total: attentionTime
+				}
+			}
+		});
+	};
+
+	const attention = new AttentionTime({ ...options, onExit });
+
+	const onScroll = (scrollDepth) => {
+		broadcast('oTracking.event', {
+			category: 'page',
+			action: 'scrolldepth',
+			meta: {
+				percentagesViewed: scrollDepth,
+				attention: attention.getAttentionTime()
+			}
```

Comment:
@chrisbrownuk are you able to shed any light on why the `page:interaction` event sets `context.whatever` and `page:scrolldepth` sets `meta.whatever`? I've never seen a `meta` property before, only `context` - are they in fact the same?

Reply:
There's no good reason for them to be different.  I suspect that they are different just because they were implemented by different people.

Reply:
I've added a note to come back to this, I won't get involved now if it's working - there are enough changes already! Thanks for your help.